node/manifest.ts

```diff
@@ -141,6 +142,10 @@ const generateManifest = ({
       excluded_patterns: excludedPattern.map(serializePattern),
     }
 
+    if ('path' in declaration) {
+      route.raw_pattern = declaration.path
```

Comment:
Not sure about the name. `path` or `pattern` would make sense here, but they're both being used already. `raw_pattern` is the best I could come up with, but I'm happy to take suggestions.

Worth noting that this is an internal field only, so not massively important.